packages/terra-form-select/src/native-select/NativeSelect.jsx

```diff
@@ -216,10 +216,10 @@ const NativeSelect = ({
     currentValue = getFirstValue(options, isFilterStyle);
     currentDisplay = getDisplay(currentValue, options, isFilterStyle, intl);
   }
-
+  const selectedLabel = intl.formatMessage({ id: 'Terra.form.select.selected' });
   const selectAttrs = {
     'aria-describedby': ariaDescribedBy,
-    'aria-label': ariaLabel,
+    'aria-label': (uncontrolledValue || value !== undefined) ? `${ariaLabel}, ${selectedLabel}` : ariaLabel,
```

Comment:
This does not seem to be accounting for internationalization properly.  Are we certain that in every language it makes sense to just add a comma between these values.  We are taking away the flexibility for the translators here.  What message are you trying to create?

Reply:
We are including selected instruction in aria-label attribute and already we have translated selected text so we are combining selected native select value as well as selected instruction, now i have removed comma between native select value and selected text. - [e5e0ac0](https://github.com/cerner/terra-core/pull/3860/commits/e5e0ac0037181db80a83493913e9bd04e58e66df)

Reply:
we have included separator on other terra components as well and has verified with Translation team. we can use the separators to enhance the screen reader response ( whenever screen reader encounters `,` it gives natural pause before reading following phrase ).